tools/block-generator/runner/run.go

```diff
@@ -151,7 +151,7 @@ func (r *Args) run() error {
 
 	err = t.Execute(f, conduitConfig)
 	if err != nil {
-		return fmt.Errorf("execute template file: %v", err)
+		return fmt.Errorf("problem executing template file: %v", err)
```

Comment:
since you're here, could you also please update %v to %w?

Reply:
hehe I shall